app/views/admins/show_procedures.html.erb

```diff
@@ -1,12 +1,11 @@
-All Procedures | <%= link_to "Pending Procedures", admins_path, method: :'get' %>
+All Procedures | <%= link_to "Back To Admin Page", admins_path() %>
```

Comment:
```suggestion
All Procedures | <%= link_to "Back To Admin Page", admins_path %>
```

Reply:
removed method: :get, and () from admins_path.